pkg/reconciler/nscert/nscert_test.go

```diff
@@ -449,12 +462,12 @@ func kubeNamespace(name string) *corev1.Namespace {
 	}
 }
 
-func kubeExcludedNamespace(name string) *corev1.Namespace {
+func kubeNamespaceWithDisableLabelValue(name, disable string) *corev1.Namespace {
```

Comment:
the name `disable` is strange here. I'd just go with `value`.

Reply:
Done